src/PrestaShopBundle/Controller/Admin/ModuleController.php

```diff
@@ -130,9 +132,8 @@ public function refreshCatalogAction(Request $request)
             $responseArray['status'] = true;
         } catch (Exception $e) {
             $responseArray['msg'] = $translator->trans(
-                'Cannot get catalog data, please try again later. Reason: '.
-                print_r($e->getMessage(), true),
-                array(),
+                'Cannot get catalog data, please try again later. Reason: %error_details%',
+                array('%error_details%' => print_r($e->getMessage(), true)),
                 'AdminModules'
```

Comment:
What is this domain? If displayed on Modules page, should be Admin.Modules.Notification